src/tests/ftest/launch.py

```diff
@@ -897,6 +901,74 @@ def archive_logs(avocado_logs_dir, test_yaml, args):
     spawn_commands(host_list, "; ".join(commands), 900)
 
 
+def send_notification(hosts, subject, msg, attachment, emails):
+    """Send email notification to provided emails with given message.
+
+    Args:
+        host (list): hosts to perform command in.
+        msg (str): message to be sent.
+        attachment (str): attachment to send on the email.
+        emails (list): list of email addresses to send message to.
+    """
+    mail_cmd = "mail -s \"{}\" -a {} {} <<< \"{}\"".format(
+        subject, attachment, ",".join(emails), msg)
+    spawn_commands(hosts, mail_cmd, 30)
+
+
+def get_log_size(test_yaml, test_file, args, size_limit=2**33):
+    """Get the size of the the directoy of the host test log files in
+    the avocado results directory and store the values in a file.
+
+    Args:
+        test_yaml (str): yaml file containing host names
+        test_file (str): the test python file
+        args (argparse.Namespace): command line arguments for this program
+        size_limit (int): limitsize in bytes that determines when to send
+            notification.
+    """
+    logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", DEFAULT_DAOS_TEST_LOG_DIR)
+    log_size_file = os.path.join(logs_dir, "log_size.log")
+    host_list = get_hosts_from_yaml(test_yaml, args)
+
+    # Create a file that contains the sizes on each test log and the logs_dir.
+    command = "du -ab -d 1 {} &> {}".format(logs_dir, log_size_file)
+    spawn_commands(host_list, command, 30)
+
+    # Check log_size.log file to see if anything goes over size_limit
+    size_info_cmd = "cat {}".format(log_size_file)
+    size_info_out = get_remote_output(host_list, size_info_cmd, 30)
+
+    # Create a dictionary of hosts for each unique return code
+    results = {code: hosts for code, hosts in size_info_out.iter_retcodes()}
+
+    # Determine if the command completed successfully across all the hosts
+    status = len(results) == 1 and 0 in results
+    if not status:
+        print("  Errors detected running \"{}\":".format(command))
+
+    # Check the command output
+    for code in sorted(results):
+        output_data = list(size_info_out.iter_buffers(results[code]))
+        print("Out data: {}".format(output_data))
```

Comment:
This doesn't seem too useful:
```
01:38:45.588  Out data: [(<ClusterShell.MsgTree.MsgTreeElem object at 0x7f39fb2613d0>, ['wolf-52']), (<ClusterShell.MsgTree.MsgTreeElem object at 0x7f39fb2d90d0>, ['wolf-123']), (<ClusterShell.MsgTree.MsgTreeElem object at 0x7f39fb2c7c50>, ['wolf-124']), (<ClusterShell.MsgTree.MsgTreeElem object at 0x7f39fb2aa6d0>, ['wolf-125']), (<ClusterShell.MsgTree.MsgTreeElem object at 0x7f39fb2f0fd0>, ['wolf-122'])]
```

Reply:
You are right, this is a print I had added for debugging.

Reply:
Will remove.